api/src/main/java/com/datastrato/gravitino/authorization/SecurableObject.java

```diff
@@ -4,21 +4,42 @@
  */
 package com.datastrato.gravitino.authorization;
 
-import com.datastrato.gravitino.annotation.Evolving;
+import com.datastrato.gravitino.annotation.Unstable;
 import javax.annotation.Nullable;
 
 /**
  * The securable object is the entity which access can be granted. Unless allowed by a grant, access
- * is denied. Gravitino organizes the securable objects using tree structure. The securable object
- * may be a catalog, a table or a schema, etc. For example, `catalog1.schema1.table1` represents a
- * table named `table1`. It's in the schema named `schema1`. The schema is in the catalog named
- * `catalog1`. Similarly, `catalog1.schema1.topic1` can represent a topic.
- * `catalog1.schema1.fileset1` can represent a fileset. `*` represents all the catalogs. If you want
- * to use other securable objects which represents all entities," you can use their parent entity,
+ * is denied. Gravitino organizes the securable objects using tree structure. <br>
+ * There are three fields in the securable object: parent, name, and type. <br>
+ * The types include 6 kinds: CATALOG,SCHEMA,TABLE,FILESET,TOPIC and METALAKE. <br>
+ * You can use the helper class `SecurableObjects` to create the securable object which you need.
+ * <br>
+ * You can use full name and type of the securable object in the RESTFUL API. <br>
+ * For example, <br>
+ * If you want to use a catalog named `catalog1`, you can use the code
+ * `SecurableObjects.ofCatalog("catalog1")` to create the securable object, or you can use full name
+ * `catalog1` and type `CATALOG` in the RESTFUL API. <br>
+ * If you want to use a schema named `schema1` in the catalog named `catalog1`, you can use the code
+ * `SecurableObjects.ofSchema("schema1")` to create the securable object, or you can use full name
+ * `catalog1.schema1` and type `SCHEMA` in the RESTFUL API. <br>
+ * If you want to use a table named `table1` in the schema named `schema1`, you can use the code
+ * `SecurableObjects.ofTable("table1")` to create the securable object, or you can use full name
+ * `catalog1.schema1.table1` and type `TABLE` in the RESTFUL API. <br>
+ * If you want to use a topic named `topic1` in the schema named `schema1`, you can use the code
+ * `SecurableObjects.ofTopic("topic1")` to create the securable object, or you can use full name
+ * `catalog1.schema1.topic1` and type `TOPIC` in the RESTFUL API. <br>
+ * If you want to use a fileset named `fileset1` in the schema named `schema1`, you can use the code
+ * `SecurableObjects.ofFileset("fileset1)` to create the securable object, or you can use full name
```

Comment:
I think it is different the API you offered, you miss the parent securable object.